staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2784,6 +2784,11 @@ type PodAffinityTerm struct {
 	// selected pods is running.
 	// Empty topologyKey is not allowed.
 	TopologyKey string `json:"topologyKey" protobuf:"bytes,3,opt,name=topologyKey"`
+	// A label query over the set of namespaces for which the term applies to.
+	// The term is applied to the union of the namespaces selected by this field
+	// and the ones listed in the `Namespaces` field.
```

Comment:
`namespaces` should be lowercase. Probably mention something similar about the union in the comment for namespaces.

You also need to say that this field is alpha level and mention the gate.

Reply:
Also mention the only valid operator

Reply:
updated.